apis/beacon/blocks/blinded_blocks.yaml

```diff
@@ -24,6 +24,9 @@ post:
             - $ref: '../../../beacon-node-oapi.yaml#/components/schemas/SignedBeaconBlock'
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Altair.SignedBeaconBlock"
             - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Bellatrix.SignedBlindedBeaconBlock"
+      application/octet-stream:
+        schema:
+          description: "SSZ serialized block bytes. Use Accept header to indicate that SSZ data is contained in the request body."
```

Comment:
Ain't the client also suppose to send a version header along with ssz request so the node knows what ssz type to use for deserialization. We had that discussion before but if I remember correctly we don't wanna put the assumption that fields will always stay in the same order.

I wonder do you actually need Accept header and if it's appropriate. Accept header is used to tell the server what content the client understands. In this case, client will automatically send content-type header which the node can use to determine if he is receiving JSON or ssz byte stream

Let me know if you need help describing header field.

Reply:
all good. ssz is fixed format for up to slot at least, but added it.